hack/testing/util.sh

```diff
@@ -325,13 +324,24 @@ function curl_es_pod_with_token() {
     shift; shift; shift;
     local args=( "${@:-}" )
     local authtoken="-H Authorization: Bearer $test_token"
-    local xff="-H X-Forwarded-For:127.0.0.1"
     oc -n $LOGGING_NS exec -c elasticsearch "${pod}" -- curl --silent --insecure "${args[@]}" \
                              $authtoken \
-                             $xff \
                              "https://localhost:9200${endpoint}"
 }
 
+function curl_es_pod_from_kibana_with_token() {
+    local pod="$1"
+    local eshost="$2"
+    local endpoint="$3"
+    local test_token="$4"
+    shift; shift; shift; shift;
+    local args=( "${@:-}" )
+    local authtoken="-H Authorization: Bearer $test_token"
+    oc -n $LOGGING_NS exec -c kibana "${pod}" -- curl --silent --insecure "${args[@]}" \
+                             $authtoken \
```

Comment:
I would skip the variable here for `-H "Authorization: Bearer ${test_token}"`. It enhance readability of the curl command as if you were typing and reading in the CLI.